oddbird/oddleventy
# Blm

## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

![](https://images.unsplash.com/photo-1596825270787-07880ab2344d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2553&q=80)

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/DXbXqluw/97-re-add-blm-page

## Description
_The commit messages say what you did; this should explain why and/or how._



## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- "alert" banner on every page
- `/community/blm/` ported from oldsite, with only minor edits
- Toggle `live: true` on/off in `content/_meta/info/alert.md` to see the alert banner turned on or off
- Check in various (esp light/dark) color schemes

## Show me
_Provide screenshots/animated gifs/videos if necessary._

![Screen Shot 2020-08-10 at 1 48 15 PM](https://user-images.githubusercontent.com/104161/89824364-3dbd5780-db10-11ea-827e-320e8dfd0cf6.jpg)
![Screen Shot 2020-08-10 at 1 48 25 PM](https://user-images.githubusercontent.com/104161/89824371-3eee8480-db10-11ea-9177-744238e2e2dd.jpg)
